pymc/model/core.py

```diff
@@ -1387,11 +1407,17 @@ def create_value_var(
 
         # Make the value variable a transformed value variable,
         # if there's an applicable transform
-        if transform is UNSET:
+
+        if default_transform is UNSET:
             if rv_var.owner is None:
-                transform = None
+                default_transform = None
             else:
-                transform = _default_transform(rv_var.owner.op, rv_var)
+                default_transform = _default_transform(rv_var.owner.op, rv_var)
+
+        if transform is UNSET:
+            transform = default_transform
+        else:
+            transform = ChainedTransform([default_transform, transform])
```

Comment:
I'm always confused about the order. If we have something like `default_transform=logodds, transform=ordered`, we want `transform.backward` to be ordered but still in logoods. I think your order is correct, but a test will be useful :)

Reply:
Yeah, it`s confused me too. I will add tests for this PR